ddtrace/contrib/flask/patch.py

```diff
@@ -323,7 +322,7 @@ def traced_start_response(status_code, headers):
             config.flask,
             method=request.method,
             url=request.base_url,
-            query=compat.to_unicode(request.query_string),
+            query=request.query_string.decode(errors="backslash"),
```

Comment:
Is this meant to be:

```suggestion
            query=request.query_string.decode(errors="backslashreplace"),
```